.github/workflows/acceptance_tests.yml

```diff
@@ -0,0 +1,147 @@
+name: Acceptance tests
+
+on:
+  pull_request:
+  schedule:
+    - cron: '0 */12 * * *'
+
+jobs:
+  test_using_builtin_python:
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ 'ubuntu-16.04', 'ubuntu-18.04', 'macos-latest', 'windows-latest' ]
+        python-version: [ '2.7.x', '3.5.x', '3.6.x', '3.7.x', '3.8.x', 'pypy2', 'pypy3' ]
```

Comment:
Why not lose all the __.x__ clutter? __python-version: ['2.7', '3.5', '3.6', '3.7', '3.8', 'pypy2', 'pypy3']__

Reply:
I am not sure how to answer, I just took that from an [example](https://github.com/actions/setup-python). Looks like `.x` can be removed.

Reply:
Please remove this clutter.